integrations/sparse/test_lucenesearcher_check_irst.py

```diff
@@ -19,7 +19,7 @@
 import unittest
 
 
-class TestLtrMsmarcoPassageIbm(unittest.TestCase):
+class TestLtrMsmarcoPassageIRST(unittest.TestCase):
```

Comment:
I think this is misnamed? There's no LTR here right?

`TestMsmarcoPassageIrst`?

Note `Irst` not `IRST` per standard naming rules.

Reply:
yes, sorry, that's a typo